docs/data.rst

```diff
@@ -171,6 +171,46 @@ Response
             ....
         ]
 
+Get FLOIP flow results for a specific form
+------------------------------------------
+Provides a list of rows of submitted data for a specific form.
+
+.. raw:: html
+
+  <pre class="prettyprint">
+  <b>GET</b> /api/v1/data/<code>{pk}</code>?content_type=application/vnd.org.flowinterop.results+json</pre>
+
+Example
+^^^^^^^^^
+::
+
+      curl -X GET https://api.ona.io/api/v1/data/210902 Accept:"application/vnd.org.flowinterop.results+json"
+
+Response
+^^^^^^^^^
+::
+
+      [
+          [ "2017-05-23T13:35:37.119-04:00", 20394823948, 923842093, "ae54d3", "female", {"option_order": ["male","female"]} ],
+          [ "2017-05-23T13:35:47.822-04:00", 20394823950, 923842093, "ae54d7", "chocolate", null ]
+      ]
+
+Get FLOIP flow results for a specific submission
+------------------------------------------
+Provides a list of rows of submitted data for a specific submission in a form.
+
+.. raw:: html
+
+  <pre class="prettyprint">
+  <b>GET</b> /api/v1/data/<code>{pk}{submission_id}</code><?content_type=application/vnd.org.flowinterop.results+json</pre>
```

Comment:
Use of the `content_type` query parameter is not supported with this PR, what is supported is using the `Accept` header in a GET request.
```
http GET /api/v1/data/{pk}/{dataid} Accept:"application/vnd.org.flowinterop.results+json"
```

Also, the `submission_id` is in the wrong position, see the above URL to confirm the correct positioning.